LibraBFT/Impl/Consensus/Types/EpochIndep.agda

```diff
@@ -528,3 +578,6 @@ module LibraBFT.Impl.Consensus.Types.EpochIndep where
     TooManySignatures    : Usize → Usize → VerifyError
     InvalidSignature     :                 VerifyError
 
+  -- TODO-1: Implement this (low priority)
+  ErrLog : Set
```

Comment:
As discussed with @cwjnkins, something like this would be preferable:
```
data ErrLog : Set where
FakeError : ErrLog
```
This is one of multiple reasons that `unit` shows up in proofs, making them less readable (another is for our `RWST` environment defined for `LBFT` (in `LibraBFT.Impl.Util.Util`).  But maybe let's leave that until after we merge this and some of @cwjnkins work on top of it.

Reply:
I was not privy to this conversation, so NO change for now.

Reply:
OK, I guess we'll keep this one for later too.